meerk40t/core/elements/elements.py

```diff
@@ -3198,7 +3198,13 @@ def list_subpath_bounds(obj):
                 result.sort(key=lambda a: a[2])
                 return result
 
+            # TODO: This is not working properly yet, so we skip this for now
```

Comment:
todo.

Reply:
Fine I will comment the complete section out as it is not working properly in cases of subpaths yet